File: practica8.py
#! /usr/bin/env python
# encoding: utf-8

# Programa que doni els primers 100 naturals en ordre
i = 0

nombre1=int(raw_input("Introduieix el primer nombre: "))
nombre2=int(raw_input("Introdueix el segon nombre: "))

for i in range (nombre1+1,nombre2,1):
	print i

File: practica8.java
import java.io.*;
//Programa que demani els dos nombres i mostri tots els que hi ha entre l'un i l'altre

public class practica8{
		public static void main(String[] args) throws IOException {
		
		int nombre1;
		int nombre2;
		String linia;
		int i;
		
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		
		System.out.print("Introdueix el primer nombre: ");
		linia = reader.readLine();
		nombre1 = Integer.parseInt(linia);
		
		System.out.print("Introdueix el segon nombre: ");
		linia = reader.readLine();
		nombre2 = Integer.parseInt(linia);
		
		for (i=nombre1+1;i<nombre2;i++) {
			System.out.println(i);
		
		}

	}
	
}
